eng/mgmt/mgmtmetadata/servicebus_resource-manager.txt

```diff
@@ -3,12 +3,13 @@ AutoRest installed successfully.
 Commencing code generation
 Generating CSharp code
 Executing AutoRest command
-cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/main/specification/servicebus/resource-manager/readme.md --csharp --version=v2 --reflect-api-versions --csharp-sdks-folder=D:\AutoRestSDK\azure-sdk-for-net\sdk
-2021-09-22 22:52:40 UTC
+cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/main/specification/servicebus/resource-manager/readme.md --csharp --version=v2 --reflect-api-versions --tag=package-2021-11 --csharp-sdks-folder=C:\SwaggerNetSdk\6\azure-sdk-for-net\sdk
+Autorest CSharp Version: 2.3.82
+2022-06-06 06:37:03 UTC
 Azure-rest-api-specs repository information
 GitHub fork: Azure
 Branch:      main
-Commit:      9b489ff217acd3bd6d62b2932e42d3c08ed4d08c
+Commit:      475e39b72894a8bdef428a472bc6d661c50032eb
 AutoRest information
 Requested version: v2
-Bootstrapper version:    autorest@1.9.1
+Bootstrapper version:    autorest@3.4.2
```

Comment:
autorest@3.4.2 is for track2, and here this pr is track 1 and which is being deprecated soon, here already have track 2 https://www.nuget.org/packages/Azure.ResourceManager.ServiceBus/1.0.0-beta.4 released

Reply:
@ArthurMa1978 , this is a management sdk, and we have been using this version of autorest for sometime now. We have used the same autorest version here as well https://github.com/Azure/azure-sdk-for-net/pull/28804. I also understand that using track 2 sdk can cause a lot of breaking changes. If we are generating the wrong way, could you please give me the command that uses the right track version. I am not sure I fully understand what you said. Thanks !